### PR TITLE
Fix elements 814 video storyboard dimensions

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

**Comment by guirenard** [comment]
:+1:

File: viewers/nuxeo-video-viewer.html [hunk 1]
```diff
@@ -38,27 +39,26 @@
...
 
       #video.hasStoryboard {
-        height: calc(100% - 85px);
+        max-height: calc(100% - 85px);
       }
 
       #storyboard {
-        display: block;
         background: var(--primary-background-color);
+        display: grid;
```
Comment (Gabez0r) [inline viewers/nuxeo-video-viewer.html]: This is a hack for a problem that I believe should be fixed on `nuxeo-document-page`: the fact that it allows content to overflow: I'd keep this as `block`:
```suggestion
        display: block;
```
And change the following on `nuxeo-document-page`:
```diff
diff --git a/elements/document/nuxeo-document-page.html b/elements/document/nuxeo-document-page.html
index ec102fe6..cba8254e 100644
--- a/elements/document/nuxeo-document-page.html
+++ b/elements/document/nuxeo-document-page.html
@@ -78,6 +78,7 @@ limitations under the License.
         @apply --layout-vertical;
         @apply --layout-flex-2;
         padding-right: 8px;
+        overflow: hidden;
       }
 
       :host([opened]) .main {
```

File: viewers/nuxeo-video-viewer.html [hunk 2]
```diff
@@ -38,27 +39,26 @@
...
         height: 85px;
```
Comment (Gabez0r) [inline viewers/nuxeo-video-viewer.html]: Also, move this to the `.thumbnail` class, to prevent it from changing on firefox.

**Comment by nmpcunha** [comment]
https://github.com/nuxeo/nuxeo-web-ui/pull/389

**Comment by Gabez0r** [comment]
👍

**Pull request merged**